cm2350/intc_exc.py

```diff
@@ -195,7 +195,7 @@ class MachineCheckPrioException(INTCException):
     __priority__ = INTC_LEVEL.MACHINE_CHECK
 
 class ResetException(INTCException):
-    __priority__ = INTC_LEVEL.MACHINE_CHECK
+    __priority__ = INTC_LEVEL.RESET
```

Comment:
I'm not opposed to this change, but want to point out that what was there previously should be the most correct, according to my original research.  I don't know what RESET level is.  I'm guessing it should always be handled first.  I'm also guessing you're right.  I just remember having this mean something specific as pertained to the fictional documentation. :)

Reply:
the difference is the exception priority, which shouldn't matter often but if we don't use the RESET priority could mean that machine check exceptions may be handled before the system reset which should have the highest priority. The exception priorities from the EREF manual  (Table 8-45. "Asynchronous Exception Priorities" and Table 8-46. "Synchronous Exception Priorities") are different from the e200 exception priorities in the e200z759C reference manual (Table 7-33. "e200z759n3 exception priorities").

Reply:
Makes sense.  Annoying differences.